controllers/academia.js

```diff
@@ -0,0 +1,259 @@
+const User = require("../modals/academia");
+const { SendEmail } = require("../common/email");
+const path = require("path");
+const JWT = require("jsonwebtoken");
+const { sendOTP } = require("../common/email");
+const JWT_SECRET_KEY = process.env.JWT_SECRET_KEY;
+
+exports.signUP = async (req, res) => {
+  try {
+    console.log(req.body);
+    const user = new User(req.body);
+    user.email = req.body.email.toLowerCase();
+
+    user.password = await User.CreateHash(user.password);
+    const oldUser = await User.findOne({
+      $or: [{ email: user.email.toLowerCase() }, { phone: user.phone }],
+    });
+    if (oldUser) {
+      res.status(400).json({
```

Comment:
use middleware fro response and error handling or use throw exception if express provide any